src/lib/backpack/costume-payload.js

```diff
@@ -0,0 +1,37 @@
+import jpegThumbnail from './jpeg-thumbnail';
+import storage from '../storage';
+
+const costumePayload = costume => {
+    // TODO is it ok to base64 encode SVGs? What about unicode text inside them?
```

Comment:
I think base64 encoding unicode should work fine. I don't see any reason why encoding SVGs this way wouldn't work.